source/program/remote_api.hpp

```diff
@@ -3,13 +3,42 @@
 #include <functional>
 #include <array>
 
+typedef std::vector<u8>* PacketBuffer;
```

Comment:
Should use `std::unique_ptr<std::vector<u8>>` instead.
This will need some changing in places that use it, I'll post some comments.